<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GNG3DISettingsWidget</class>
 <widget class="QWidget" name="GNG3DISettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #333333;
color: #ffffff;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="separate_camera_control">
     <property name="text">
      <string>Separate camera control</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="skip_rendering">
     <property name="text">
      <string>Skip rendering</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dynamicLod">
     <property name="text">
      <string>Dynamic Level of Detail</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mode_edge">
     <property name="text">
      <string>Age rendering</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="seed"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Point size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="point_size_node">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Signal color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="signal_color_node"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Winner color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="winner_color_node"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Winner 2 color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="winner2_color_node"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>No winner color</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="no_winner_color_node">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Line color</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="line_color_edge"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Line size</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="line_size_edge"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Min Detail</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="minDetail"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Min Detail Distance</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="minDetailDistance"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Max Detail</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="maxDetail"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Max Detail Distance</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="maxDetailDistance"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="apply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
